### ISSUE TITLE
OpenTelemetry tracing instead of a custom solution?

### ISSUE THREAD

**Issue by fprochazka** [getsentry/sentry-java#1348]
Hi,
I'd like to ask if you'd be willing to consider building on top of an open standard for tracing?

In the past, I've littered my code with Newrelic libs and annotations, then AWS X-Ray and now I stand before the decision if I want to try out Sentry tracing. And I really don't want to do that, if I'd have to vendor-lock myself into another lib, that I'd have to then manually remove if I decide to switch backends for storing traces.

With OpenTelemetry support, I would be able to switch tracing backends without having to change my code.

* https://opentelemetry.io/docs/java/

**Comment by fprochazka** [comment]
I've just noticed you're working on a Sentry exporter https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/sentryexporter, that's great!

**Comment by bruno-garcia** [comment]
We have interest in OpenTelemetry, even made some contributions to their SDKs, but their spec only went GA (traces) recently, and we've been evolving our product so far without necessarily being dependant/blocked by it. We plan to explore more going forward but to this point, the way to use of Sentry's performance product is through the Sentry SDKs.

We have the exporter that you found but our individual SDKs are not dependant on OTel for multiple reasons, including the fact the data model doesn't map 1 to 1. And the data that the OTel integrations will capture, that you can pipe to Sentry through the collector won't be as rich as if you use the Sentry integration.

**Comment by smeubank** [comment]
Hi @fprochazka 

You can check out this project https://github.com/orgs/getsentry/projects/39/views/1 here and get links to ongoing work in the form PoC PRs and developer specs for OTEL transformation to use with Sentry. Would be happy to hear your thoughts. Java isn't immediately on the plan at the moment but we are always happy to get someone to give feedback in the early days.

For what its worth there is an exporter, but it was a small project, and we do not promote it with confidence. But I do know a handful of companies are using it.

**Comment by marandaneto** [comment]
@adinauer maybe this one can be closed since you've created https://github.com/getsentry/sentry-java/issues/2327

**Comment by adinauer** [comment]
Yeah, closing this in favor of #2327
